app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'authorid'
+  has_many :comments, class_name: 'Comment', foreign_key: 'postid'
+  has_many :likes, class_name: 'Like', foreign_key: 'postid'
+  after_save :update_post
+
+  def update_post
+    counter = Post.count('authorid')
+    User.update(postscounter: counter)
+  end
+
+  def recents_comments
+    Comment.limit(5).order(created_at: :desc)
+  end
+end
```

Comment:
- [ ] Please the post model should have a method that returns the 5 most recent comments for a given post.  

def recent_comments
  comments.order('created_at Desc').limit(5)
end

Reply:
Ok im going to change it